src/ReadLoader.cpp

```diff
@@ -467,6 +467,14 @@ void ReadLoader::processFastqFileThreadFunction(size_t threadId)
             continue;
         }
 
+        // Skip if the q scores have an obvious palindromic characteristic
+        span <char> scores(scoresBegin, scoresEnd);
+        if (isPalindromic(scores)){
```

Comment:
You have to add a new category of discarded reads. The category you are incrementing here refers to reads that were discarded because they were too short. When you create that new category, a few things need to happen. Just follow what we do for short reads as a pattern. That will include storing in `AssemblerInfo` and writing to `AssemblySummary.html` and `AssemblySummary.json`.